src/pi/pi_controller.c

```diff
@@ -126,10 +126,17 @@ static void dma_pi_write(struct pi_controller* pi)
     dram = (uint8_t*)pi->ri->rdram.dram;
     rom = pi->cart_rom.rom;
 
-    for(i = 0; i < longueur; ++i)
-    {
-        dram[(dram_address+i)^S8] = rom[(rom_address+i)^S8];
-    }
+	if (((dram_address | rom_address | longueur) & 3) != 0)
```

Comment:
I suggest a tiny comment after the if statement `check word-alignment` (or maybe it's obvious).